website/content/en/docs/concepts/scheduling.md

```diff
@@ -307,7 +319,7 @@ spec:
 ```
 See [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) in the Kubernetes documentation for details.
 
-## Topology Spread
+### Topology Spread
```

Comment:
Do we want to talk about preferred topology spread and create an alert block like you did over in the affinity section? You could keep it short and sweet and link it out like a proposed up above

Reply:
Added.